CHANGES.rst

```diff
@@ -2,7 +2,7 @@ Changes
 =======
 
 Version master (UNRELEASED)
----------------------------
+***************************
```

Comment:
I'm surprised this would be necessary... According to [sphinx docs](https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings), we are using headings OK, i.e. CHANGES is underlined via `=` (as in Sample H3), so Versions should be underlined via `-` as in Sample H4.

Is this something that the theme changed? If so, perhaps we should customise it there...

Reply:
I've changed all the titles to match sphinx docs and it works.

Reply:
Interesting :wink: Looks nicer. BTW some of the rst page titles are still using different styles, e.g. README uses hashes, CONTRIBUTING uses asterisks, AUTHORS is only equal-sign-underlined. Would it be possible to harmonise that?

```console
$ head -4 *.rst
==> AUTHORS.rst <==
Authors
=======

The list of contributors in alphabetical order:

==> CHANGES.rst <==
*******
Changes
*******


==> CONTRIBUTING.rst <==
************
Contributing
************


==> README.rst <==
############
REANA-Server
############
```

Reply:
Yes, all of them are harmonized now except the README's that should be h1.